### ISSUE TITLE
`test_scale_down_rgw` does not run properly over external mode clusters

### ISSUE THREAD

**Issue by MeridianExplorer** [red-hat-storage/ocs-ci#4307]
Currently, the test begins by looking for RGW deployments on the cluster - 
```
rgw_deployment = pod.get_deployments_having_label(
            constants.RGW_APP_LABEL, defaults.ROOK_CLUSTER_NAMESPACE
        )[0]
```
However, modes running in external mode do not have the deployment, leading the test to fail with an IndexError (because of the `[0]`).

I believe the tests should be skipped in external mode - @ebenahar?

**Comment by MeridianExplorer** [comment]
@ebenahar WDYT?

**Comment by MeridianExplorer** [comment]
Addressed in #5147
